device/dell/x86_64-dellemc_s5248f_c3538-r0/DellEMC-S5248f-P-25G/sesto-1.json

```diff
@@ -0,0 +1,256 @@
+{
+  "lanes": [
+    {
+      "index": 200,
+      "local_lane_id": 0,
+      "system_side": true,
+      "tx_polarity": 0,
+      "rx_polarity": 0,
+      "line_tx_lanemap": 0,
+      "line_rx_lanemap": 0,
+      "line_to_system_lanemap": 0,
```

Comment:
are these three above only applicable for line side lanes? if yes, then can you remove it for system side lanes?

Reply:
The lanemaps are also not currently used and are a placeholder as recommended by the original SAI_Gearbox_API_Proposal documentation.